Presentation.Web/app/components/it-system/it-system-catalog.view.html

```diff
@@ -39,54 +39,87 @@ <h3>Opsummering af ændringer ifm. flytning af "IT-system anvendelse" til nyt IT
-                        Id: {{affectedContracts.contract.id}}
-                    </div >
-                    <div>
-                        <div class="migrationDivBorders migrationDivPadding">
-                            IT-systemet er omfattet af kontrakten <strong> {{affectedContracts.systemAssociatedInContract ? 'Ja' : 'Nej' }}</strong>
-                        </div>
-                        <div class="migrationDivPadding" ng-if="affectedContracts.affectedInterfaceUsages.length > 0" ng-class="{'migrationDivBorders': affectedContracts.interfaceExhibitUsagesToBeDeleted.length > 0}">
-                            "Anvender" snitfladerelation, der flyttes
-                            <ul>
-                                <li ng-repeat="interfaceToBeMoved in affectedContracts.affectedInterfaceUsages">{{interfaceToBeMoved.name}} ( Id: {{interfaceToBeMoved.id}} )</li>
-                            </ul>
-                        </div>
-                        <div class="migrationDivPadding" ng-if="affectedContracts.interfaceExhibitUsagesToBeDeleted.length > 0">
-                            "Udstiller" snitfladerelation, der slettes
-                            <ul>
-                                <li ng-repeat="interfaceToBeDeleted in affectedContracts.interfaceExhibitUsagesToBeDeleted">{{interfaceToBeDeleted.name}} ( Id: {{interfaceToBeDeleted.id}} )</li>
-                            </ul>
-                        </div>
-                    </div>
+                <div class="panel panel-default" ng-if="systemCatalogVm.migrationReportDTO.affectedContracts.length > 0">
+                    <table class="table">
+                        <tr>
+                            <th>Id</th>
+                            <th>Navn</th>
+                        </tr>
+                        <tr ng-repeat="affectedContracts in systemCatalogVm.migrationReportDTO.affectedContracts">
+                            <td class="idTableDataCellFormat">{{affectedContracts.id}}</td>
+                            <td>{{affectedContracts.name}}</td>
+                        </tr>
+                    </table>
+                </div>
+
+                <h4 ng-if="systemCatalogVm.migrationReportDTO.affectedRelations.length > 0">Følgende system relationer berøres:</h4>
+                <div class="panel panel-default" ng-if="systemCatalogVm.migrationReportDTO.affectedRelations.length > 0">
+                    <table class="table">
+                        <tr>
+                            <th>
+                                Relation
+                                <br />
+                                Fra
+                            </th>
+                            <th>
+                                <br />
+                                Til
+                            </th>
+                            <th>Relationsbeskrivelse</th>
+                            <th>Snitflade der fjernes fra relationen</th>
+                            <th>Kontrakt</th>
```

Comment:
Nu hvor jeg ser det, så er det her sted det eneste sted hvor frekvensen ikke er med. Den skal nok med for at informationen er fyldestgørende. Den er også valgfri ligesom kontrakten